docs/_reference/auth.md

```diff
@@ -478,4 +478,25 @@ methods:
         }
       }
       ```
+  - signature: auth.updateProfile(profile)
+    id: updateProfile
+    generator: true
+    description: Updates the user's basic profile information.
+    arguments:
+      - name: profile
+        required: true
+        type: A {displayName: nullable string, photoURL: nullable string} object
+        description: The profile's displayName and photoURL to update.
+    example: |
+      ```javascript
+      function* updateProfileSaga(profile) {
+        try {
+          yield call(rsf.auth.updateProfile, profile);
```

Comment:
Could you give an example of what `profile` should look like?
I guess something like:

```js
function* updateProfileSaga(
  try {
    yield call(rsf.auth.updateProfile, {
      displayName: "blah blah",
      photoURL: "skldqk"
    });
             
    yield put(updateProfileSuccess());
  }
  ...
```